pkg/domain/infra/tunnel/containers.go

```diff
@@ -193,6 +193,10 @@ func (ic *ContainerEngine) ContainerRm(ctx context.Context, namesOrIds []string,
 		}
 		reports := make([]*entities.RmReport, 0, len(ctrs))
 		for _, c := range ctrs {
+			if opts.Force && opts.Timeout != nil {
+				// Ignore errors, containers.Remove will catch them
+				containers.Stop(ic.ClientCtx, c.ID, &containers.StopOptions{Timeout: opts.Timeout})
```

Comment:
Why are we doing this? Remove already stops containers if force is specified?

Reply:
This just strikes me as potentially buggy - we ignore all errors that could occur with stopping the container... Seems like it would be a lot safer to add a Timeout arg to the libpod RemoveContainer function?

Reply:
There is some nasty plumbing to do.  We will still try to stop the container within the remove stage if it is not stopped, so I don't believe there is any risk.

Reply:
This allows us to stop them first without a timeout of 0.

Reply:
This can be removed now I think